scripts/evaluation/driver.py

```diff
@@ -96,9 +97,12 @@ def main():
     client = Client(base_url=args.eval_api_url, verify=False)  # noqa: S501
 
     if "localhost" not in args.eval_api_url:
-        with open(args.eval_api_token_file, mode="r", encoding="utf-8") as t_f:
-            token = t_f.read().rstrip()
-        client.headers.update({"Authorization": f"Bearer {token}"})
+        if path.isfile(args.eval_api_token_file) == True: 
```

Comment:
this is to accommodate GHA secret retrieval, as we cannot pass secret under a file WRT GHA